converter/biostudies.py

```
@@ -100,6 +101,9 @@ def __init__(self):
         }
 
     def convert(self, hca_project: dict, additional_attributes: dict = None) -> dict:
+        if hca_project.get('releaseDate') is None:
+            self.__set_release_date(hca_project)
```

I'm not sure if this is a good idea. I worry that we may publish things that aren't supposed to be published yet. This release date should come from the user.

---

ok, but what if it is not set? Shall I just return with an error?

---

Yeah, it might be better to raise that as exception then we let the user edit the project's release date.

---

We might want to make it mandatory in the first place

---

I am going to think about it how to make it better. Tx for the suggestion. 🤔

---

I discussed it with BioStudies and it is not mandatory, so I am going to change this code.

---

@aaclan-ebi Pushed the changed code and a test